Components/Bites/src/OgreApplicationContext.cpp

```diff
@@ -379,6 +379,8 @@ NativeWindowPair ApplicationContext::createWindow(const Ogre::String& name, Ogre
 #if OGRE_PLATFORM == OGRE_PLATFORM_EMSCRIPTEN
     SDL_GL_CreateContext(ret.native);
     miscParams["currentGLContext"] = "true";
+    miscParams["parentWindowHandle"] = ropts["parentWindowHandle"].currentValue;
```

Comment:
drop this part: ropts should not contain parentWindowHandle and we should respect whatever is in miscParams